scripts/uberenv/packages/umpire/package.py

```diff
@@ -276,6 +277,20 @@ def hostconfig(self, spec, prefix, py_site_pkgs_dir=None):
             cfg.write(cmake_cache_entry("CMAKE_CXX_FLAGS_RELWITHDEBINFO", reldebinf_flags))
             cfg.write(cmake_cache_entry("CMAKE_CXX_FLAGS_DEBUG", debug_flags))
 
+        #Configuration to enable developer benchmarks (i.e. No-Op)
+        if "+dev_benchmarks" in spec:
+            cfg.write("#------------------{0}\n".format("-" * 60))
+            cfg.write("# Developer Benchmarks\n")
+            cfg.write("#------------------{0}\n\n".format("-" * 60))
+            
+            cfg.write(cmake_cache_option("ENABLE_DEVELOPER_BENCHMARKS", True))
+            cfg.write(cmake_cache_option("ENABLE_BENCHMARKS", True)) #Enable BLT GoogleBenchmark support
+            cfg.write(cmake_cache_string("CMAKE_BUILD_TYPE", spec.variants['build_type'].value)) 
```

Comment:
This is already set here: https://github.com/LLNL/Umpire/blob/529a6e7ec4112b211208c0166d21ed929bf19edf/scripts/uberenv/packages/umpire/package.py#L209

Reply:
Bah Humbug you're right